tests/Request/AdjustmentRequestTest.php

```diff
@@ -1,32 +0,0 @@
-<?php
-
-namespace AllDigitalRewards\Tests;
-
-use AllDigitalRewards\RewardStack\Adjustment;
-use PHPUnit\Framework\TestCase;
-
-class AdjustmentRequestTest extends TestCase
-{
-    protected $uniqueId;
-    protected $adjustmentRequest;
-
-    protected function setUp(): void
-    {
-        $this->uniqueId = uniqid();
-        $this->adjustmentRequest = new Adjustment\AdjustmentRequest($this->uniqueId);
-    }
-
-    public function testGetHttpEndpoint()
-    {
-        $expectedUrl = '/api/user/' . $this->uniqueId . '/adjustment';
```

Comment:
Not suggesting these were good testcases; but what were the problems with it again? It looks like it's testing the URL and that the return object of a response is the expected hydrate response entity?